calrissian/version.py

```diff
@@ -1,3 +1,21 @@
+import pkg_resources  # part of setuptools
+
+
+def package_version(package_name):
+    try:
+        pkg = pkg_resources.require(package_name)
+        return pkg[0].version
+    except pkg_resources.DistributionNotFound:
+        return 'unknown'
+
+
+def cwltool_version():
+    return package_version('cwltool')
+
+
+def calrissian_version():
+    return package_version('calrissian')
+
 
 def version():
-    return 'Calrissian-dev'
+    return 'calrissian {} (cwltool {})'.format(calrissian_version(), cwltool_version())
```

Comment:
I tried `calrissian --version` and it just printed help. I expected to see version info like `cwltool` does.

Reply:
Moved to #43. This happens because calrissian checks for its required args and exits with help before passing the args to cwltool's `main()`. `--version` isn't checked until inside cwltool's `main()`, so I need to duplicate that handling in calrissian.